src/time.cr

```diff
@@ -140,7 +140,15 @@ struct Time
     new(seconds: seconds + offset, nanoseconds: nanoseconds, kind: Kind::Local)
   end
 
-  def self.new(year, month, day, hour = 0, minute = 0, second = 0, nanosecond = 0, kind = Kind::Unspecified)
+  def self.new(year, month, day, hour = 0, minute = 0, second = 0, *, millisecond = 0, kind = Kind::Unspecified)
+    unless 0 <= millisecond <= 999_999
```

Comment:
shouldn't milliseconds should be between 0 and 999.
Otherwise we are speaking of microseconds. Which, could be other overload if needed.

Reply:
Oh right, I knew I was missing something here... thanks!